src/components/Sparklines.js

```diff
@@ -0,0 +1,54 @@
+import { LineChart, Line } from 'recharts';
+
+const staticData = [
```

Comment:
Can we remove the dummy data from this file? Since we have it in the rankings component, we should just be taking it in as a parameter